CHANGELOG.md

```diff
@@ -8,9 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [6.3.0] - 2023-10-??
 
 - Added: During generation, if no alternatives have a non-zero weight, try weighting by how many additional Nodes are reachable.
+- Added: Changelog now shows the date an update was publishe
```

Comment:
I prefer to leave the wording to native speakers but `publishe` looks like a typo to me.

Reply:
I would say something like `The changelog now shows the date for each release`

Reply:
Changed the entry to dyceron's suggestion 👍 and yes that was a typo